test/e2e_env/kubernetes/meshtrafficpermission/api.go

```diff
@@ -64,12 +73,62 @@ spec:
           version: v1
       default:
         action: DENY
-`)(env.Cluster)).To(Succeed())
+`, Config.KumaNamespace))(env.Cluster)).To(Succeed())
 
 		// then
 		mtps, err = env.Cluster.GetKumactlOptions().KumactlList("meshtrafficpermissions", meshName)
 		Expect(err).ToNot(HaveOccurred())
 		Expect(mtps).To(HaveLen(1))
-		Expect(mtps[0]).To(Equal("mtp1.kuma-system"))
+		Expect(mtps[0]).To(Equal(fmt.Sprintf("mtp1.%s", Config.KumaNamespace)))
+	})
+
+	It("should deny creating policy in the non-system namespace", func() {
+		// given no MeshTrafficPermissions
+		mtps, err := env.Cluster.GetKumactlOptions().KumactlList("meshtrafficpermissions", meshName)
+		Expect(err).ToNot(HaveOccurred())
+		Expect(mtps).To(HaveLen(0))
+
+		// when
+		err = k8s.KubectlApplyFromStringE(
+			env.Cluster.GetTesting(),
+			env.Cluster.GetKubectlOptions(), `
+apiVersion: kuma.io/v1alpha1
+kind: MeshTrafficPermission
+metadata:
+  name: mtp1
+  namespace: default
+  labels:
+    kuma.io/mesh: meshtrafficpermission-api
+spec:
+  targetRef:
+    kind: MeshService
+    name: backend
+  from:
+    - targetRef:
+        kind: Mesh
+      default:
+        action: ALLOW
```

Comment:
nit: this test is about blocking policies with non-system namespace. Can we just have one item in the from list? The rest is irrelevant.